requirements-upload.txt

```diff
@@ -0,0 +1,9 @@
+certifi==2018.8.24
```

Comment:
Could you reduce this to what's actually required, not just a `pip freeze`?

Reply:
Probably doable. Looks to me like it's literally just twine and up to date pip.